free-1.md

```diff
@@ -10,41 +10,63 @@ _What need is met by your idea? Who are you helping? What is that person's
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+It makes text adventure games.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Generally speaking, the user would describe the layout of the world, what is in it and where, and what actions the player is allowed to take. I conjecture (based entirely upon my own intuition) that many people would find it easier to create a world by describing it and letting the computer do the rest rather than by telling the computer how to set everything up. I would want to ask as many people as possible whether this is true for them before actually committing to this
+design decision. I do expect it to be true, though, because what I just described is pretty much exactly how normal (non-interactive) fiction works.
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+The person interacting with the language might not be the person interacting with the program itself. In any case, whoever is using the program plays the text adventure, typing at a command line sort of thing in the usual style for a text adventure. I think it might be best for this DSL to be compiled so that any actual errors in the code could be communicated to the writer at compile time. Any errors that would occur at runtime would be the result of the player trying to do something
+they're not allowed to do. The program could tell the player that it doesn't recognize a particular word they used in their command, because "I don't know what that means," while traditional, isn't very helpful.
+
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to place a bunch of rooms in a world and specify their relation to each other. It should be easy to set the player's starting location. It should be easy to create items that the player can pick up. It should be easy to specify what a player can do with an item (either as a property of the item or as a property of something else that interacts with the item). I think it should be impossible to define new data types, which would mean that items and the like would need
+to be instances of already-defined data types. I'll add a caveat that it should only be impossible if this is implemented as an internal DSL; if it's external, it should just be difficult to do so to encourage users to work within the language but allow for functionality that the language doesn't have but maybe should.
+
```

Comment:
hmm, are you sure this is true on a global level? Its easy for us to just assume our experience (mudd) generalizes but especially related to people's technical abilities, the generalization usually fails.